nightwatch/components/chipWithIcon.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import ChipWithIcon from '../../src/components/ChipWithIcon';
+import { Error } from '../../src/icons';
+import GlobalStyles from '../../src/components/GlobalStyles';
+
+export default {
+  title: 'Chip With Icon Component',
+  component: ChipWithIcon
+};
+
+export const ChipWithIconComponent = () => (
+  <>
+    <GlobalStyles />
+    <ChipWithIcon  icon={<Error data-testid="chip-icon" />}>
+      Sample Chip
+    </ChipWithIcon>
+  </>
+);
+
+export const ChipWithIconComponentWithTransformText = () => (
+  <>
+    <GlobalStyles />
+    <ChipWithIcon transformText icon={<Error data-testid="chip-icon" />}  >
+      Sample Chip
+    </ChipWithIcon>
+  </>
+);
+
+ChipWithIconComponent.test = async (browser, { component }) => {
+  browser.expect(component).to.be.visible;
+  browser.expect(component).text.to.equal('Sample Chip');
+  browser.expect.element('[data-testid="chip-icon"]').to.be.present;
+};
+
+ChipWithIconComponentWithTransformText.test = async (browser, { component }) => {
+  browser.expect(component).to.be.visible;
+  browser.expect.element('[data-testid="chip-icon"]').to.be.present;
+  browser.expect(component).text.to.equal('sample chip');
```

Comment:
`transformText` converts `sample chip` to `Sample Chip`

Reply:
do i need to add other test with "sample chip" and expect "Sample Chip" ??

Reply:
or just change the values and keep it two tests ??

Reply:
when enable the transformText it did not convert sample chip to Sample chip
as you can see
![image](https://github.com/nightwatchjs/html-reporter/assets/76897266/8b922ec4-6976-4deb-a422-42ea8c53d949)